lib/core/migrate/migrate.js

```diff
@@ -557,6 +557,22 @@ const migrations = [
 			}
 			Storage.set('RESmodules.newCommentCount.subscriptions', subscriptions);
 		},
+	}, {
+		versionNumber: '5.9.1-userTagger-refresh',
+		async go() {
+			let tags = await Storage.wrapPrefix('tag.', (): any => ({})).getAll();
+			for (const tag of Object.values(tags)) {
+				if (tag.votes > 0) tag.votesUp = tag.votes;
+				if (tag.votes < 0) tag.votesDown = -tag.votes;
+				Reflect.deleteProperty(tag, 'votes');
```

Comment:
If you wrote this before I changed the rule to only forbid `\.call` and `\.apply`, you can change it back to `delete`. If you prefer this that's fine too.